index.js

```diff
@@ -51,7 +50,12 @@ const chainIdMapping = Object.entries({
 	return memo;
 }, {});
 
-const getNetworkFromId = ({ id }) => chainIdMapping[id];
+/** @type {(obj: {id: number} | number) => number} */
+const getNetworkFromId = (obj) => {
+	const id = typeof obj === 'number' ? obj : obj.id;
+	chainIdMapping[id];
```

Comment:
Dont forget to return this please :)

Reply:
lol I'm incapable of writing a single line of working code without typescript it appears
```suggestion
	return chainIdMapping[id];
```